README.md

```diff
@@ -21,6 +21,8 @@
     - [Pushing commits](#pushing-commits)
     - [Pulling branches](#pulling-branches)
     - [Pull requests](#pull-requests)
+  - [Necessary Hardware](#necessary-hardware)
+  - [Network Map](#network-map-1)
```

Comment:
@WowMuchDoge Add a section that says what steps need to be followed for that vision service. Include the actual `vision.service` file and how to reload it, restart the service, and check its status